cpp/src/parquet/metadata.cc

```diff
@@ -1684,6 +1702,17 @@ class RowGroupMetaDataBuilder::RowGroupMetaDataBuilderImpl {
       total_compressed_size += column_builders_[i]->total_compressed_size();
     }
 
+    const auto& sorting_columns = properties_->sorting_columns();
+    if (!sorting_columns.empty()) {
+      std::vector<format::SortingColumn> thrift_sorting_columns(sorting_columns.size());
```

Comment:
What about adding the bi-directional thrift conversion in a single place as here: https://github.com/apache/arrow/blob/main/cpp/src/parquet/thrift_internal.h#L79-L101 ? That would be consistent with others.

Reply:
Will do it